ui/test.html

```diff
@@ -21,7 +21,7 @@
   <head>
     <meta charset="utf-8">
     <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,900|Source+Code+Pro' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="app.css">
```

Comment:
I assume this was just pointing to the wrong location?  There's nothing that moves the stylus build output in this commit.